tools/benchmark/src/index.ts

```diff
@@ -39,14 +39,17 @@ export {
 	CustomBenchmarkArguments,
 } from "./Configuration";
 export { benchmark, benchmarkMemory, IMemoryTestObject, MemoryTestObjectProps } from "./mocha";
-export { prettyNumber, geometricMean, Stats } from "./ReporterUtilities";
+export { prettyNumber, geometricMean } from "./ReporterUtilities";
 export { BenchmarkReporter } from "./Reporter";
+export { Phase, runBenchmark } from "./runBenchmark";
 export {
+	Stats,
 	BenchmarkData,
 	BenchmarkError,
+	MemoryBenchmarkStats,
 	BenchmarkResult,
+	MemorySampleData,
+	MemoryTestData,
 	isResultError,
```

Comment:
I think we don't want to export these extra types, they are all "implementation details" of the runners for runtime benchmarks or memory benchmarks, so it shouldn't be necessary to expose them outside the package.

Reply:
Yes, they are only used by `MemoryTestRunner` to avoid unsafe type eslint error. I have disable esline rule and remove them from here.